libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 mod error;
-mod messages;
+pub mod messages;
```

Comment:
>If this module is now public, we should remove the Cm prefix

Do you think this should be call `Message`, `ConnectionManagerMessage`, or something else? Personally I'd advocate for `Message`.

>We should also consider being consistent on whether or not we pluralize the module

The convention we seem to have adopted for splinter is to `error.rs` for error modules and `messages.rs` for message mondules. So the names are consistent with other modules in project atm. FWIW I think this discussion is out of this PRs scope.

Reply:
Since this PR was originally written, I think the changes required is that module shouldn't be public, and the connector should be tranforming those from the internal message to the appropriate `Result<T, ConnectionManagerError>`.  That probably goes beyond the scope of this PR.